eslint-plugin-expensify/no-negated-variables.js

```diff
@@ -2,30 +2,62 @@ const _ = require('underscore');
 const lodashGet = require('lodash/get');
 const message = require('./CONST').MESSAGE.NO_NEGATED_VARIABLES;
 
+const BANNED_SUBSTRINGS = [
+    'not',
+    'isnot',
+    'cannot',
+    'shouldnot',
+    'cant',
+    'dont',
+];
+
+const NOTABLE_EXCEPTIONS = [
+    'notification',
+    'notch',
+    'note',
+    'notable',
+    'notion',
+    'notice',
+];
+
+// Declared up front to suppress eslint no-use-before-define
+// Mutually recursive functions are a valid exception to this rule. See https://github.com/eslint/eslint/issues/12473
+let isNegatedVariableName;
+
 /**
  * @param {String} string
  * @returns {Boolean}
  */
-function isFalsePositive(string) {
-    return _.some(['notification', 'notch'], falsePositive => string.toLowerCase().includes(falsePositive));
-}
+const isFalsePositive = (string) => {
```

Comment:
NAB, not sure why this needs to be changed to an arrow function.

Reply:
It doesn't really – I was just making it consistent with the one above (which also doesn't necessarily need to be an arrow function). It just looked weird to me to do:

```javascript
isNegatedVariableName = function(string) {
}
```

I guess that's fine but I'm just more used to seeing arrow functions used like this. So it was just muscle memory I guess 🤷

Reply:
>I'm just more used to seeing arrow functions used like this

Got it. This rule could maybe be interpreted as "don't use arrow functions in modules"...

https://github.com/Expensify/App/blob/main/STYLE.md#functions-1

Up to you though 🙃